src/shared/form_fields/member.js

```diff
@@ -105,6 +104,24 @@ const required =
   , 'news_type'
   ]
 
+const read_only_user =
```

Comment:
This is fine, though I would probably define it as `read_only.concat([ 'due_date' ])`